packages/components/Separator/src/SeparatorTokens.android.tsx

```diff
@@ -0,0 +1,11 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { SeparatorTokens } from './Separator.types';
+
+export const defaultSeparatorTokens: TokenSettings<SeparatorTokens, Theme> = (t: Theme) =>
```

Comment:
Is this change needed for iOS as well?

Reply:
Okay it seems like tokenization is up to individual platforms to complete, so I'm guessing not